fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -23,10 +24,10 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :command,
                                        env_name: "FL_CARTHAGE_COMMAND",
-                                       description: "Carthage command (one of `build`, `bootstrap`, `update`)",
+                                       description: "Carthage command (one of `build`, `bootstrap`, `update`, `archive`)",
                                        default_value: 'bootstrap',
                                        verify_block: proc do |value|
-                                         UI.user_error!("Please pass a valid command. Use one of the following: build, bootstrap, update") unless %w(build bootstrap update).include? value
```

Comment:
oops looks like this comment didn't get fixed up in the squash. please take a look.